services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java

```diff
@@ -24,7 +24,7 @@
    * @param user                    {@link RouteOptions#user()}
    * @param profile                 {@link RouteOptions#profile()}
    * @param coordinates             {@link RouteOptions#coordinatesList()}
-   * @param accessToken             {@link RouteOptions#accessToken()}
+   * @param accessToken             the access token
```

Comment:
Why is this changing?

Reply:
Broken Javadocs, I was practicing the [The Boy Scout Rule](https://biratkirat.medium.com/step-8-the-boy-scout-rule-robert-c-martin-uncle-bob-9ac839778385) 🧹

Reply:
Oh never mind, RouteOptions does not have accessToken https://github.com/mapbox/mapbox-java/pull/1276

Reply:
Yeah, not anymore, some leftovers from https://github.com/mapbox/mapbox-java/pull/1276